workspace.jsonc

```diff
@@ -365,18 +360,14 @@
       "teambit.dependencies/dependency-resolver": {
         "policy": {
           "dependencies": {
-            "@teambit/legacy": "-",
-            // TODO: remove this later
-            "core-js": "3.8.3"
+            "@teambit/legacy": "-"
           },
           "devDependencies": {
             "@types/mocha": "5.2.7",
             "@teambit/legacy": "-"
           },
           "peerDependencies": {
-            "react": "16.13.1",
-            "@teambit/legacy": "1.0.49",
-            "react-dom": "16.13.1"
```

Comment:
this is a bit dangerous - many of the components in bit are using the aspect env not the react env